BitwardenShared/Core/Auth/Services/AuthService.swift

```diff
@@ -769,6 +779,11 @@ class DefaultAuthService: AuthService { // swiftlint:disable:this type_body_leng
         try await authAPIService.resendEmailCode(resendEmailModel)
     }
 
+    func resendNewDeviceOtp() async throws {
+        guard let resendNewDeviceOtpModel else { throw AuthError.unableToResendNewDeviceOtp }
+        try await authAPIService.resendNewDeviceOtp(resendNewDeviceOtpModel)
+    }
```

Comment:
🤔 Should the `resendNewDeviceOtpModel` be cleared at some point if this succeeds or if some particular error is thrown?

Reply:
Good catch! Thank you!

Reply:
I'll address this in the next PR since it has part of the code that makes sense to clear this value out.